cmd/node/main.go

```diff
@@ -157,118 +165,127 @@ VERSION:
 	//  /debug/pprof/trace?seconds=5 (CPU trace) -> being a trace, can be analyzed with: go tool trace
 	// Usage: go tool pprof http(s)://ip.of.the.server/debug/pprof/xxxxx
 	profileMode = cli.BoolFlag{
-		Name:  "profile-mode",
-		Usage: "Boolean profiling mode option. If set to true, the /debug/pprof routes will be available on the node for profiling the application.",
+		Name: "profile-mode",
+		Usage: "Boolean profiling mode option. If set to true, the /debug/pprof routes will be available on the node" +
+			" for profiling the application.",
 	}
 	// skIndex defines a flag that specifies the 0-th based index of the private key to be used from initialNodesSk.pem file
 	skIndex = cli.IntFlag{
-		Name:  "sk-index",
-		Usage: "Private key index specifies the 0-th based index of the private key to be used from initialNodesSk.pem file.",
+		Name: "sk-index",
+		Usage: "Private key `index` specifies the 0-th based index of the private key to be used from the PEM file which" +
+			" contains multiple private keys.",
 		Value: 0,
 	}
 	// gopsEn used to enable diagnosis of running go processes
 	gopsEn = cli.BoolFlag{
 		Name:  "gops-enable",
-		Usage: "Enables gops over the process. Stack can be viewed by calling 'gops stack <pid>'",
+		Usage: "If set, will enable gops over the process. Stack can be viewed by calling 'gops stack <pid>'.",
 	}
 	// storageCleanup defines a flag for choosing the option of starting the node from scratch. If it is not set (false)
 	// it starts from the last state stored on disk
 	storageCleanup = cli.BoolFlag{
 		Name:  "storage-cleanup",
-		Usage: "If set the node will start from scratch, otherwise it starts from the last state stored on disk",
+		Usage: "If set the node will start from scratch, otherwise it starts from the last state stored on disk.",
```

Comment:
Boolean option for starting the Node with clean storage. If set to true, the Node will empty its storage (is this true) before starting, otherwise it will start from the last state stored on disk.